src/plugins/d3/renderer/hooks/useSeries/types.ts

```diff
@@ -155,6 +156,7 @@ export type PreparedLineSeries = {
             };
         };
     };
+    dashStyle?: DashStyle;
```

Comment:
Prepared series should not contain optional fields if possible. You need to set the default value to avoid unnecessary checks further